From 106390e2d520e3eea2795c146b971d55c3c59794 Mon Sep 17 00:00:00 2001
From: Harwood <1222752+Harwood@users.noreply.github.com>
Date: Thu, 6 Sep 2018 12:59:03 -0500
Subject: [PATCH] Problem: link to getting_started does not resolve

This prevents the link from working when README.md is rendered

Solution: fix the link
---
 README.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/README.md b/README.md
index 4ae5b67..7324f4e 100644
--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@
 <p align="center">
   [
     <a href="https://github.com/sit-fyi/issue-tracking/releases"><b>Download</b></a> |
-    <a href="doc/getting_started.md"><b>Getting Started</b></a> |
+    <a href="web/documentation/getting_started.md"><b>Getting Started</b></a> |
     <a href="#questions-bug-reports-etc"><b>Issues & Merge Requests</b></a>
   ]
 </p>
@@ -34,7 +34,7 @@ $ sit init
 $ sit-web
 ```
 
-That's it. Now you can check out the [Getting Started](doc/getting_started.md) guide.
+That's it. Now you can check out the [Getting Started](web/documentation/getting_started.md) guide.
 
 <center>
 <p align="center">
-- 
2.18.0

